pages/index.mdx

```diff
@@ -50,13 +67,14 @@ As your service grows, GOV.UK PaaS lets you instantly:
 
 ## Resilient and secure
 
-GOV.UK PaaS is designed to meet the needs of government teams working on public-facing services:
+GOV.UK PaaS is designed to meet the needs of government teams working on public-facing services by:
 
--   UK hosted
--   We aim for 99.95% uptime for your applications and [you can view historical performance](https://status.cloud.service.gov.uk/)
--   It offers 24-hour platform-level support
--   It's possible to test an application in the [trial period free of charge](/pricing)
--   The technology runs on an IA-accredited infrastructure, so times for app accreditation are generally faster
+-   being hosted domestically within the UK
```

Comment:
Dublin isn't in the UK

Reply:
Perhaps we should say hosted on the "British isles"? Or cut to the chase and say Dublin.

Reply:
UK _or_ Dublin ?

Reply:
being hosted domestically within the UK by default or Dublin by request? 

We don't start new orgs in Dublin

Reply:
yep I think the latter works nicely